crates/codegen/tests/fixtures/match_literal.snap

```diff
@@ -0,0 +1,197 @@
+---
+source: crates/codegen/tests/yul.rs
+expression: output
+input_file: tests/fixtures/match_literal.fe
+---
+{
+  function match_bool(e) -> ret {
+    let v0 := 0
+    switch e
+      case 1 {
+        v0 := 1
+      }
+      default {
+        v0 := 2
+      }
+  }
+}
```

Comment:
This should assign `ret := v0` (or equivalent). (Same problem in all match tests)

Reply:
Good catch. Pushed another commit to fix that